graph/patterns/enums.md

```diff
@@ -100,7 +100,7 @@ but it is also open for future scenarios:
 
 Additionally speaking, depending on the situation, a nullable enum can very likely be avoided by adding a `none` member.
 
-#### Flag Enums or Collection of Enums
+## Flag Enums or Collection of Enums
 
 In case an enum can have multiple values at the same time the tentation is to model the property as a collection of Enums:

```

Comment:
Did you observe this behavior in practice? may be we need to describe only preferred option like flagged enums?